packages/big-design/src/components/Timepicker/Timepicker.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { forwardRef, memo, Ref, useMemo } from 'react';
+
+import { createLocalizationProvider, getTimeIntervals } from '../../utils';
+import { Select } from '../Select';
+interface Props {
+  error?: React.ReactNode;
+  label?: React.ReactChild;
```

Comment:
Can we add some tests around passing these values into the component?

Reply:
Done.